apps/web/lib/features/team/team-outstanding-notifications.tsx

```diff
@@ -37,45 +38,45 @@ export function TeamOutstandingNotifications() {
 	);
 }
 
-function UserOutstandingNotification({ outstandingPlans, user }: { outstandingPlans: IDailyPlan[]; user?: IUser }) {
+const UserOutstandingNotification = memo(function UserOutstandingNotification({
+	outstandingPlans,
+	user
+}: {
+	outstandingPlans: IDailyPlan[];
+	user?: IUser;
+}) {
 	const t = useTranslations();
 
-	// Notification will be displayed 6 hours after the user closed it
-	const REAPPEAR_INTERVAL = 6 * 60 * 60 * 1000; // 6 hours in milliseconds;
```

Comment:
@CREDO23 / @Cedric921 so we had before requirement to do something with those 6 hours, why we removing that now? Did you find task that was talked about those 6 hours? Can we clarify with @KostyaEver on that? Because PR talks about a bit other things and seems it REMOVES that feature now... So I wonder how we build it before and now removing :D

Reply:
not sure about '6 hours' feature, but the general idea was 
1. if user or team members have tasks in Outstanding - show the Notification bar 'How many tasks remain'
2. if user goes directly to the Outstanding tab without clicking on the Notification, it also should disappear
3. The Notification should appear ONLY one time a day normally when login first time at that day and if user closed/checked should disappear


Currently, we don't have at all that notification bar https://github.com/ever-co/ever-teams/issues/3060
@evereq @CREDO23 @Cedric921

Reply:
> @CREDO23 / @Cedric921 so we had before requirement to do something with those 6 hours, why we removing that now? Did you find task that was talked about those 6 hours? Can we clarify with @KostyaEver on that? Because PR talks about a bit other things and seems it REMOVES that feature now... So I wonder how we build it before and now removing :D

Before, notifications could be displayed after 6 hours !
Now, as @KostyaEver described in the issue description, notifications should be displayed by next day. @evereq

Reply:
Now issue that it does not display Notification at all @CREDO23 
and @evereq has asked what action led to the case that currently the functionality does not work at all?

Reply:
@CREDO23 why you "invented" new code if you could just change 6 hours to 24 hours and that's it!?